package.json

```diff
@@ -54,7 +54,16 @@
         }
       },
       "prettier"
-    ]
+    ],
```

Comment:
just released a new config version, could you try `^0.0.26` before going this route? if that version doesn't work I'm fine merging and I'll fix in another PR.